cuda/test/matrix/csr_kernels.cpp

```diff
@@ -232,5 +233,17 @@ TEST_F(Csr, ConjugateTransposeIsEquivalentToRef)
                     static_cast<ComplexMtx *>(trans.get()), 0.0);
 }
 
+TEST_F(Csr, ConvertToDenseIsEquivalentToRef)
+{
+	set_up_apply_data(std::make_shared<Mtx::cusparse>());
+
+	auto dense_mtx = gko::matrix::Dense<>::create(ref);
+	auto ddense_mtx = gko::matrix::Dense<>::create(cuda);
```

Comment:
`const`

Reply:
Same here, it should not be `const`. I guess I was too fixated on setting everything to `const`...